packages/manager/apps/web/client/app/hosting/website-coach/website-coach.html

```diff
@@ -0,0 +1,63 @@
+<div class="container-fluid px-5 mt-5">
+    <div class="row">
+        <div class="col-md-4">
+
+            <!-- Screenshot visualizer -->
+            <div class="website-coach-visualizer">
+                <img class="website-coach-visualizer__computer"
+                    src="/assets/images/website-coach-computer.png"
+                    alt="{{$ctrl.attachedDomain}}">
+                <div class="website-coach-visualizer__screenshot"
+                    data-ng-style="::{
+                        'background-image': 'url(\'data:image/png;base64,' + $ctrl.screenshot + '\')'
+                    }">
+                </div>
+            </div>
+        </div>
+        <div class="col-md-8">
+            <oui-back-button
+                data-on-click="$ctrl.goBack()"
```

Comment:
```suggestion
                data-on-click=":: $ctrl.goBack()"
```

Reply:
OTB with a function ?
I'll check for the others